src/extension/index.ts

```diff
@@ -20,6 +21,10 @@ import { Store } from './store';
 import * as Telemetry from './telemetry';
 import { update, updateChannelConfigSection } from './update';
 import { UriRebaser } from './uriRebaser';
+import * as fs from 'fs';
```

Comment:
You can act directly on the `promises` object by importing from `'fs/promises'` instead